aws_lambda_powertools/utilities/parser/models/sqs.py

```diff
@@ -1,25 +1,33 @@
-from datetime import datetime
+from datetime import datetime, timezone
 from typing import Dict, List, Optional, Sequence, Type, Union
 
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
 
 from aws_lambda_powertools.utilities.parser.types import Literal
 
 
 class SqsAttributesModel(BaseModel):
     ApproximateReceiveCount: str
     ApproximateFirstReceiveTimestamp: datetime
-    MessageDeduplicationId: Optional[str]
-    MessageGroupId: Optional[str]
+    MessageDeduplicationId: Optional[str] = None
+    MessageGroupId: Optional[str] = None
     SenderId: str
     SentTimestamp: datetime
-    SequenceNumber: Optional[str]
-    AWSTraceHeader: Optional[str]
+    SequenceNumber: Optional[str] = None
+    AWSTraceHeader: Optional[str] = None
+
+    # Validator to normalize the ApproximateFirstReceiveTimestamp and SentTimestamp fields
+    # Converts the provided timestamp value to a UTC datetime object
+    # See: https://github.com/pydantic/pydantic/issues/6518
+    @validator("ApproximateFirstReceiveTimestamp", "SentTimestamp", pre=True)
+    def coerce_timestamp(cls, value):
+        date_utc = datetime.fromtimestamp(int(value) / 1000, tz=timezone.utc)
+        return date_utc
```

Comment:
The Pydantic project will revert the behavior back to what we had in v1, so we can remove this validator after that.

Issue: https://github.com/pydantic/pydantic/issues/6518

Reply:
They fixed this bug in v2.0.3 - https://github.com/pydantic/pydantic/releases/tag/v2.0.3.

We have removed this extra code now.